cpp/test/IceGrid/deployer/AllTests.cpp

```diff
@@ -520,8 +520,8 @@ allTests(Test::TestHelper* helper)
 
     cout << "testing service configuration... " << flush;
     obj = TestIntfPrx(comm, "IceBox1-Service1@IceBox1.Service1.Service1");
-    test(obj->getProperty("Service1.Type") == "standard");
-    test(obj->getProperty("Service1.ServiceName") == "Service1");
+    test(obj->getProperty("Service1Data.Type") == "standard");
```

Comment:
It's not immediately clear why you added this Data suffix.

If it is necessary, does it mean this update could break user configuration?

Reply:
The tests was setting unknown properties on an OA and disabling warnings.